Android.bp

```diff
@@ -96,3 +102,8 @@ cc_binary {
     static_libs: ["libtinyalsav2"],
     cflags: ["-Werror"],
 }
+
```

Comment:
Why do we need subdirs?

Reply:
removed, I was under the impression that we need to add subdirs to internal Android.bp files to be picked